app/main.py

```diff
@@ -14,4 +14,21 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    phrase = phrase.lower()
+    letter = letter.lower()
+
+    occurrences = 0
+
+    for char in phrase:
+        if char == letter:
+            occurrences += 1
+
+    return occurrences
```

Comment:
The solution is ok, but can you try to rewrite it using just 1 line?

Reply:
done